RELEASE/scripts/autoscend.ash

```diff
@@ -4764,12 +4771,6 @@ void print_header()
 
 boolean doTasks()
 {
-	if(get_property("_casualAscension").to_int() >= my_ascensions())
```

Comment:
I don't think this is safe to remove yet - casual ascensions are not something I want people accidentally running without knowing what they're getting into.

Reply:
change made 
2db7fb3